jenkins/ci.suse.de/templates/cloud-mkcloud-job-upgrade-disruptive-ha-mariadb-template.yaml

```diff
@@ -0,0 +1,37 @@
+- job-template:
+    ### The reason why this upgrade job is "disruptive" is the default cinder backend
+    name: 'cloud-mkcloud{version}-job-upgrade-disruptive-ha-mariadb-{arch}'
+    node: cloud-trigger
+
+    triggers:
+      - timed: '32 22 * * *'
+
+    logrotate:
+      numToKeep: -1
+      daysToKeep: 7
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            TESTHEAD=1
+            cloudsource=develcloud{previous_version}
+            upgrade_cloudsource=develcloud{version}
+            ### 3 nodes for HA cluster, 2 compute nodes
+            nodenumber=6
```

Comment:
3 + 2 is 5 not 6, and your defining the lonely node below, so 5 should be enough here.

Also in cloud-mkcloud8.yaml nodenumber has already been defined as 5, so we could get away with:
`nodenumber={nodenumber}`
So it can be defined and modified outside the template.

Reply:
Oh, you are right! I've changed the comment (I had "1 for lonelynode" there before) but forgot to modify the nodenumber....